[BUG] [GUI] chrome-sandbox is owned by root and has mode 4755

I have this error on my fresh install Debian Buster. I followed the installation guide (from source code), but i have this error when i try to launch gui

`[5017:0512/191633.734238:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/farmer/chia-blockchain/chia-blockchain-gui/node_modules/electron/dist/chrome-sandbox is owned by root and has mode 4755.`
`/home/farmer/chia-blockchain/chia-blockchain-gui/node_modules/electron/dist/electron exited with signal SIGTRAP`
`npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chia-blockchain@ electron: 'electron '.
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chia-blockchain@ electron script.`

--- a/build_scripts/build_linux_rpm.sh
+++ b/build_scripts/build_linux_rpm.sh
@@ -50,7 +50,6 @@ echo "npm build"
 npm install
 npm audit fix
 npm run build
-chmod 4755 node_modules/electron/dist/chrome-sandbox
 LAST_EXIT_CODE=$?
 if [ "$LAST_EXIT_CODE" -ne 0 ]; then
 	echo >&2 "npm run build failed!"
@@ -86,6 +85,11 @@ if [ "$REDHAT_PLATFORM" = "x86_64" ]; then
 	# Disables build links from the generated rpm so that we dont conflict with other packages. See https://github.com/Chia-Network/chia-blockchain/issues/3846
 	# shellcheck disable=SC2086
 	sed -i '1s/^/%define _build_id_links none\n%global _enable_debug_package 0\n%global debug_package %{nil}\n%global __os_install_post \/usr\/lib\/rpm\/brp-compress %{nil}\n/' "$NODE_ROOT/lib/node_modules/electron-installer-redhat/resources/spec.ejs"
+	
+	# Use attr feature of RPM to set the chrome-sandbox permissions
+	# adds a %attr line after the %files line
+	# The location is based on the existing location inside spec.ej
+	sed -i '/^%files/a %attr(4755, root, root) /usr/lib/<%= name %>/chrome-sandbox' "$NODE_ROOT/lib/node_modules/electron-installer-redhat/resources/spec.ejs"
 
 	# Updates the requirements for building an RPM on Centos 7 to allow older version of rpm-build and not use the boolean dependencies
 	# See https://github.com/electron-userland/electron-installer-redhat/issues/157

--- a/build_scripts/build_linux_rpm.sh
+++ b/build_scripts/build_linux_rpm.sh
@@ -50,6 +50,7 @@ echo "npm build"
 npm install
 npm audit fix
 npm run build
+chmod 4755 node_modules/electron/dist/chrome-sandbox
 LAST_EXIT_CODE=$?
 if [ "$LAST_EXIT_CODE" -ne 0 ]; then
 	echo >&2 "npm run build failed!"

--- a/build_scripts/build_linux_rpm.sh
+++ b/build_scripts/build_linux_rpm.sh
@@ -85,7 +85,7 @@ if [ "$REDHAT_PLATFORM" = "x86_64" ]; then
 	# Disables build links from the generated rpm so that we dont conflict with other packages. See https://github.com/Chia-Network/chia-blockchain/issues/3846
 	# shellcheck disable=SC2086
 	sed -i '1s/^/%define _build_id_links none\n%global _enable_debug_package 0\n%global debug_package %{nil}\n%global __os_install_post \/usr\/lib\/rpm\/brp-compress %{nil}\n/' "$NODE_ROOT/lib/node_modules/electron-installer-redhat/resources/spec.ejs"
-	
+
 	# Use attr feature of RPM to set the chrome-sandbox permissions
 	# adds a %attr line after the %files line
 	# The location is based on the existing location inside spec.ej

